apps/prairielearn/src/question-servers/calculation.js

```diff
@@ -162,7 +162,10 @@ module.exports.render = questionFunctionExperiment(
   calculationSubprocess.render,
 );
 
-module.exports.getFile = questionFunctionExperiment(
+// Note that the difference in naming between `file` and `getFile` is intentional.
+// The external interface of questions uses `file`, but the internal interface
+// uses `getFile`.
+module.exports.file = questionFunctionExperiment(
```

Comment:
This interface must conform to what is expected by `question.js`:

https://github.com/PrairieLearn/PrairieLearn/blob/e7df5841b5cbf12ad2b41d5fa7a06b8ddbcd99b6/apps/prairielearn/src/lib/question.js#L152